docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -74,18 +75,61 @@
 
 ## Python
 
-This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Ethereum network.
+This example uses [Web3.py](https://web3py.readthedocs.io/en/stable/) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Kovan testnet.
 
 ```python Kovan
 {% include 'samples/PriceFeeds/PriceConsumerV3.py' %}
 ```
 
+{% endmarkdown %}
+
 <div class="row cl-button-container">
     <div class="col-xs-12 col-md-12">
     <a href="https://repl.it/@alexroan/GetLatestPriceWeb3PY" class="cl-button--ghost python-tracked">Run this Python example</a>
     </div>
 </div>
 
+{% markdown %}
+
+## Golang
+
+This example uses [go-ethereum](https://github.com/ethereum/go-ethereum) to retrieve feed data from the [ETH / USD feed](https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331) on the Kovan testnet.
+In order to interact with a smart contract in go, you need to:
+
+1. Generate the [Contract ABI](https://docs.soliditylang.org/en/latest/abi-spec.html) from a solidity source file using [solc](https://docs.soliditylang.org/en/latest/using-the-compiler.html). In this case, we are going to download the [`AggregatorV3Interface`](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol) and generate an ABI called `AggregatorV3Interface.abi`:
+
+```shell 
+wget https://raw.githubusercontent.com/smartcontractkit/chainlink/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol
+solc --abi AggregatorV3Interface.sol -o .
+```
+
+2. Convert `AggregatorV3Interface.abi` to a `Go` file `aggregator_v3_interface.go`. Ensure that [Geth](https://geth.ethereum.org/docs/install-and-build/installing-geth) is installed so you can use `abigen`. This source code generator allows to convert Ethereum contract definitions into easy to use, compile-time type-safe Go packages.
+
+```shell
+abigen --abi=AggregatorV3Interface.abi --pkg=aggregator_v3_interface --out=aggregator_v3_interface.go
+```
+
+3. Import the package. You can find the whole example with all the source files [here](https://replit.com/@hophop/chainlink-price-feed-golang?v=1)
```

Comment:
Suggestion: Should be use gists that are more durable? replit is great but not sure how persistent the examples will be? Also may be more consistent for us to use gists as we use them elsewhere?

Reply:
the sources files are under [my own repo](https://github.com/aelmanaa/chainlink-price-feed-golang) but I can move them to a [devrel repo](https://github.com/smartcontractkit/smart-contract-examples) ? The main advantage with replit is the "run" button. Thus, allowing readers to easily run the example in their browser.

Reply:
@zeuslawyer I put the example [here](https://github.com/smartcontractkit/smart-contract-examples/pull/5) . Let's correct/complete then merge on the smart-contract-examples repo before finishing this PR